utils/linker/index.js

```diff
@@ -13,31 +13,14 @@ const CHECKSUM_OFFSET_MAX_MIN = 1;
 const VALID_VERSION = 1;
 const DELIMITER = "*";
 
-/**
- * Return the key value pairs
- * @param {string} value
- * @return {?Object<string, string>}
- */
-function parseLinker(value) {
-  const linkerObj = parseLinkerParamValue(value);
-  if (!linkerObj) {
-    return null;
-  }
-  const { checksum, serializedIds } = linkerObj;
-  if (!isCheckSumValid(serializedIds, checksum)) {
-    return null;
-  }
-  return deserialize(serializedIds);
-}
```

Comment:
This `parseLinker` fn is being used in **analytics.js**
In `setAnonymousId` fn.

Reply:
We haven't deleted any code. The methods have been ordered as per their declaration and usage.

Reply:
My bad. Missed the below declarations.